tensorflow/python/saved_model/model_utils/export_utils.py

```diff
@@ -242,7 +242,7 @@ def get_temp_export_dir(timestamped_export_dir):
   """
   (dirname, basename) = os.path.split(timestamped_export_dir)
   temp_export_dir = os.path.join(
-      compat.as_bytes(dirname), compat.as_bytes('temp-{}'.format(basename)))
+      compat.as_bytes(dirname), compat.as_bytes('temp-' + compat.as_str(basename)))
```

Comment:
nit: to keep things consistent with the rest of the file, can this use `\.format` instead of string concatenation?

Reply:
made the proposed change in a subsequent commit. thanks!